File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
       mp = {}

       for i in range(len(nums)):
          c = target-nums[i];

          if c in mp:
            return [mp[c],i];
          mp[nums[i]]=i;
    
       return []

File: Data Structures & Algorithms/jump-game/submission-1.java
class Solution {
    public boolean canJump(int[] nums) {
        if(nums.length==1 && nums[0]==0)return true;
        int i=0;
        while(i<nums.length){
            
            if(nums[i]==0){
                return false;
            }
            i = i + nums[i];
            System.out.println(i);
            if(i>=nums.length-1){
                return true;
            }
        }

        return i>=nums.length;
    }
}
